assets/src/dashboard/images/bookmark-fill.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="14" height="18" viewBox="0 0 14 18" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
These are all icons, right? Let's rename the folder then.

Reply:
Not exactly. In the other PR, the logo will be part of this directory. It could be an icon. I'm fine with it either way. Or we can have both icons and images.